core/runtime/jsruntime.rs

```diff
@@ -481,7 +512,7 @@ pub struct PollEventLoopOptions {
 impl Default for PollEventLoopOptions {
   fn default() -> Self {
     Self {
-      wait_for_inspector: true,
+      wait_for_inspector: false,
```

Comment:
Default is false because we mostly poll the event loop with false and this cleans up a lot of code

Reply:
I think in most cases it doesn't make any difference, but I'm fine with this change.

Reply:
It broke a few tests when I was porting it over and using ::default() everywhere, but I think I'd like to circle back on this later on as we might be able to move it out of poll_event_loop somehow (not sure how yet)